brooklyn-tosca-transformer/src/main/java/org/apache/brooklyn/tosca/a4c/brooklyn/ToscaPlanToSpecTransformer.java

```diff
@@ -239,6 +289,41 @@ private void setLocationsOnSpecs(Map<String, EntitySpec<?>> specs,
         }
     }
 
+    public boolean isABrooklynPolicy(String policyType){
+        Class clazz;
+        CatalogItem catalogItem = CatalogUtils.getCatalogItemOptionalVersion(this.mgmt, policyType);
+        if (catalogItem != null) {
+            clazz = catalogItem.getCatalogItemJavaType();
+        } else {
+            try {
+                clazz = Class.forName(policyType);
+            } catch (ClassNotFoundException e) {
+                return false;
+            }
+        }
+        return Policy.class.isAssignableFrom(clazz);
+    }
+
+    private String getPolicyType(GenericPolicy policy){
+        String type=null;
+        if(policy.getType()!=null){
+            type = policy.getType();
+        } else if( policy.getData().containsKey("type")) {
```

Comment:
This is pretty minor and won't stop me merging this - prefer whitespace around operators `if (condition)` and `x != null` rather than `if(..)` and `x!=null`, etc.